assets/scripts/wplf-form.js

```diff
@@ -26,6 +26,7 @@
 
       fetch(ajax_object.ajax_url  + '?action=wplf_submit', {
         method: "POST",
+        credentials: ajax_object.ajax_credentials || 'omit', // if it's not defined, omit. 
```

Comment:
If user uses the shortcode outside of post content, they have to include the script & localisation manually. That means they won't have a 'ajax_credentials' defined in their localisation array, and existing behaviour would change.